src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -40,6 +41,9 @@
          * to think about number formatting issues.
          */
         Utils.forceJVMLocaleToUSEnglish();
+
+        // Set config factory to know about the configuration options that we have.
```

Comment:
It wouldn't work with extended classes - that was something I was trying to fix.  @lbergelson and I brainstormed and came up with a much more elegant way to do this initialization.  Briefly, by abstracting away the `owner` interface for getting/creating configs and using one in `ConfigUtils` we can do this check at config creation time.  So moving forward users will use the `ConfigUtils::create` and `ConfigUtils::getOrCreate` to create configurations of any type.